Blog/view/logIn.php

```diff
@@ -0,0 +1,56 @@
+<?php
+
+session_start();
+
+//include('Blog/include/functions.php');
+include('Blog/include/loginFunctions.php');
+					$Errors = array();
```

Comment:
This code is heavily indented. Might make things more readable if you cleaned that up a bit.

Maybe you know this, but you can just select the whole block of code and hit shift+tab to un-indent the whole block.